<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>391</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="tickBtn">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>391</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,3,0,1,0">
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="1" column="3">
     <widget class="QLineEdit" name="portEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>重新开始</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ipEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>port:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tickBtn</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>ipEdit</tabstop>
  <tabstop>portEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>tickBtn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
